index.html

```diff
@@ -7,65 +7,76 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Akshar:wght@500&family=Hind:wght@300&family=Lobster&family=Reenie+Beanie&family=Source+Sans+Pro:wght@200&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="index.css">
-    <script src="index.js"></script>
+<link rel="stylesheet" href="index.css">
+<script src="index.js"></script>
 <title>Personal Portfolio</title>
 </head>
 <body>
-    <div class="navigation">
-        <div class="logo">SHUJA</div>
-        <a id="home" href="#welcome-section">Home</a>
-        <a id="work" href="#projects">My Work</a>
-        <a id="contact" href="#contact">Contact</a>
-    </div>
-    <div class="first-section">

```

Comment:
- [ ]  Instead of using divs for sections,  there should be HTML <section>

Reply:
- [ ] [OPTIONAL] Same here, please include an explanation,  a `why`, for each requested change.